frontend/src/metabase/dashboard/components/DashboardActions.jsx

```diff
@@ -53,16 +60,19 @@ export const getDashboardActions = (
             </a>
           </div>
           <div>
-            <a
-              className={extraButtonClassNames}
-              data-metabase-event={"Dashboard;Sharing"}
-              onClick={() => {
-                self.refs.popover.close();
-                onEmbeddingClick();
-              }}
-            >
-              {t`Sharing and embedding`}
-            </a>
+            <DashboardSharingEmbeddingModal
+              additionalClickActions={() => self.refs.popover.close()}
```

Comment:
This seems gross but I don't know what else to do.

Separately, my JS-fu isn't strong enough to understand why `={self.refs.popover.close}` didn't work and I had to wrap it in a function.

Reply:
I think this is fine. I didn't look at the code, but I'm guessing `close` isn't bound to `popover`. 
```js
const  a = {b: function() { console.log(this === a); }}

let f;
f = a.b;
f() // prints false
f = () => a.b()
f() // prints true
f = a.b.bind(a)
f() // prints true
```